app/assets/javascripts/discourse/tests/integration/components/site-header-test.js

```diff
@@ -135,4 +141,47 @@ module("Integration | Component | site-header", function (hooks) {
     await waitUntil(() => getProperty() === "60px", { timeout: 100 });
     assert.strictEqual(getProperty(), "60px");
   });
+
+  test("arrow up/down keys move focus between the tabs", async function (assert) {
+    this.currentUser.set("redesigned_user_menu_enabled", true);
+    await render(hbs`<SiteHeader />`);
+    await click(".header-dropdown-toggle.current-user");
+    let activeTab = query(".menu-tabs-container .btn.active");
+    assert.strictEqual(activeTab.id, "user-menu-button-all-notifications");
+
+    await triggerKeyEvent(document, "keydown", "ArrowDown");
+    let focusedTab = document.activeElement;
+    assert.strictEqual(
+      focusedTab.id,
+      "user-menu-button-replies",
+      "pressing the down arrow key moves focus to the next tab towards the bottom"
+    );
+
+    await triggerKeyEvent(document, "keydown", "ArrowDown");
+    await triggerKeyEvent(document, "keydown", "ArrowDown");
+    await triggerKeyEvent(document, "keydown", "ArrowDown");
+    await triggerKeyEvent(document, "keydown", "ArrowDown");
+    await triggerKeyEvent(document, "keydown", "ArrowDown");
+
+    focusedTab = document.activeElement;
+    assert.ok(
+      focusedTab.href.endsWith("/u/eviltrout/preferences"),
```

Comment:
Just curious, is there no id or class we can use this identify the element here?

Reply:
Yeah there isn't any unique identifier for this element, but we will replace this element/link with a button soon and it'll have an ID then and I will update this test.